src/CaptureHigherEd/LaravelJira/Models/Issue.php

```diff
@@ -11,15 +11,8 @@ final class Issue implements ApiResponse
     const PROJECT = 'project';
     const DESCRIPTION = 'description';
     const ISSUETYPE = 'issuetype';
-
-    const CUSTOMFIELD_PARTNERWEBSITE = 'customfield_10056';
-    const CUSTOMFIELD_ENGAGEURL = 'customfield_10057';
-    const CUSTOMFIELD_POPULATIONSIZE = 'customfield_10060';
-    const CUSTOMFIELD_POPULATIONTYPES = 'customfield_10061';
-    const CUSTOMFIELD_POPULATIONNAMES = 'customfield_10062';
-    const CUSTOMFIELD_REQUESTTYPE = 'customfield_10026';
-    const CUSTOMFIELD_CLIENTNAME = 'customfield_10064';
-    const CUSTOMFIELD_PHYSICALADDRESS = 'customfield_10070';
```

Comment:
These are used in our other repos so make sure you update that code to match.

Reply:
I will!